src/main/java/ch/bfh/bti7081/Presenter/HomePresenter.java

```diff
@@ -26,4 +39,35 @@ public HomePresenter(){
     public List<Doctor> getDoctors() {
         return doctorQuerier.getAll();
     }
+
+    /**
+     * gets a specific doctor
+     * @param doctorId
+     * @return Doctor
+     */
+    public Doctor getDoctor(ObjectId doctorId){
+        return doctorQuerier.get(doctorId);
+    }
+
+    /**
+     * gets the patient by the Id of one of it's reports
+     * @param reportId
+     * @return
+     */
+    public Patient getPatientByReport(ObjectId reportId){
```

Comment:
presenter should not contain filtering. Please move to querier

Reply:
Will do